cmd/run_controller/main.go

```diff
@@ -27,6 +27,7 @@ func init() {
 
 	flag.IntVar(&burst, "burst", 10, "burst for RESTClient")
 	flag.IntVar(&qps, "qps", 5, "QPS for RESTClient")
+	flag.IntVar(&threadiness, "threadiness", 2, "threadiness for run-controller")
```

Comment:
Not a good parameter description. It's clear that this value is for the run controller, what else? Describe the meaning!

Reply:
Is it better now?